### ISSUE TITLE
ProjectTool : Requesting to install .NET 8 runtime where it should be bundled with one

### ISSUE THREAD

**Issue by stuckie** [YoYoGames/GameMaker-Bugs#6012]
### Description

Filing from conversations in Discord.
Please remember that conversations in Discord, or any community forum, social network, etc... do not replace bug reporting.
This is being added to track this, only because it's serious enough to warrant it.
If a bug is not reported, we can't know about it to fix it.

It looks like the ProjectTool compile has missed it's self-contained flag, and is requesting the .NET 8 runtime installed on users machines.
All our tools should be self-contained builds that do not require the user to specifically install a .NET runtime.

### Steps To Reproduce

1. Start GameMaker
2. Attempt to open/import older project
3. See the issue

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.600 (Betas)

### Which platform(s) are you having trouble building this project for?

Windows

### Are you running GameMaker from inside your Steam library?

None

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by stuckie** [comment]
For marking as duplicates, the problem runtime is 2024.600.0.577

**Comment by Emc1923** [comment]
Verified fixed as of  IDE v2024.6.0.157  Runtime v2024.6.0.205.

Error received with 2024.600.0.577 was:
```
Framework: 'Microsoft.NETCore.App', version '8.0.0' (x64)
.NET location: C:\Program Files\dotnet\

The following frameworks were found:
  6.0.26 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]

Learn more:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=8.0.0&arch=x64&rid=win-x64&os=win10
```
